cuegui/cuegui/Utils.py

```diff
@@ -416,8 +416,19 @@ def getResourceConfig(path=None):
 ################################################################################
 
 def getFrameLogFile(job, frame):
-    """Get the log file associated with a frame."""
-    return os.path.join(job.data.log_dir, "%s.%s.rqlog" % (job.data.name, frame.data.name))
+    """Get the log file associated with a frame. Return path based on the
+    current OS path using Constants.LOG_ROOT_OS to translate paths."""
+    my_os = platform.system().lower()
+    job_os = job.data.os.lower()
+
+    log_dir = job.data.log_dir
+    if my_os != job_os and \
+            my_os in cuegui.Constants.LOG_ROOT_OS and \
+            job_os in cuegui.Constants.LOG_ROOT_OS:
+        log_dir = log_dir.replace(cuegui.Constants.LOG_ROOT_OS[job_os],
```

Comment:
Actually it strikes me that `replace` might cause some unintended changes to the path, depending on what folks are using as their roots. Maybe we could do the substitution using trim?